syntax = "proto2";

package BackupPB;

message RestoreResponse {
	enum Status {OK=0; FAIL=1; }
	required Status status = 1;
}
